packages/store-ui/src/ProductSummary/Container.tsx

```diff
@@ -0,0 +1,27 @@
+/** @jsx jsx */
+import { FC } from 'react'
+import { jsx } from 'theme-ui'
+
+import LocalizedLink from '../LocalizedLink/index'
+
+interface Props {
+  to: string
+  variant: string
+}
+
+const ProductSummaryContainer: FC<Props> = ({ children, to, variant }) => (
+  <LocalizedLink
+    state={{ fromSummary: true }}
+    to={to}
+    sx={{
+      variant: `productSummary.${variant}.container`,
+      textDecoration: 'none',
+      color: 'text',
+      flexGrow: 1,
+    }}
```

Comment:
this cannot be sx, because it is in the store-ui, if it will not be mandatory for everyone. Reminder: sx is override for variant.